pkg/checks/checks.go

```diff
@@ -157,18 +232,10 @@ out:
 	}
 
 	// Deployments
-	dList, err := appclient.Deployments("").List(metav1.ListOptions{})
-	if err != nil {
-		return err
-	}
-
-	for _, d := range dList.Items {
-		if !checkNamespace(d.Namespace) {
-			continue
-		}
-		log.Info(fmt.Sprintf("checking %s %q", d.Kind, d.Name))
+	for idx, app := range deploymentWhitelist {
+		log.Info(fmt.Sprintf("checking  deployment[%d]: [%s] %s", idx, app.Namespace, app.Name))
```

Comment:
2 spaces

Reply:
Fixed

Reply:
Still outstanding?

Reply:
Also log.Infof instead of log.Info+fmt.Sprintf